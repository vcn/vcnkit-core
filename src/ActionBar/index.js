import styled from 'styled-components';

export default styled.div`
    display:         flex;
    justify-content: flex-end;

    min-height: 3.25rem;
    box-sizing: border-box;
    padding:    .5rem 1rem .5rem 1.5rem;

    & > * {
        margin: 0 0 0 .5rem;
    }
`;
