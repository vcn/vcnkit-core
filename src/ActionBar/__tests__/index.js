import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import ActionBar from '../';

describe('ActionBar', () => {
    test('renders', () => {
        const tree = renderer.create(
            <ActionBar>
                <button>Button 1</button>
                <button>Button 2</button>
            </ActionBar>
        ).toJSON();
    
        expect(tree).toMatchSnapshot();
    });
});
