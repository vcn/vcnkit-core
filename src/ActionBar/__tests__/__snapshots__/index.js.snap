// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActionBar renders 1`] = `
.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: end;
  -ms-flex-pack: end;
  -webkit-justify-content: flex-end;
  justify-content: flex-end;
  min-height: 3.25rem;
  box-sizing: border-box;
  padding: .5rem 1rem .5rem 1.5rem;
}

.c0>* {
  margin: 0 0 0 .5rem;
}

<div
  className="c0 c1"
>
  <button>
    Button 1
  </button>
  <button>
    Button 2
  </button>
</div>
`;
